aksel.nav.no/website/components/website-modules/OutdatedAlert.tsx

```diff
@@ -18,23 +18,25 @@ export default function OutdatedAlert() {
           Det er over 1 år siden innholdet ble revidert. Vi kan ikke være helt
           sikre på hvor nøyaktig artikkelen er lenger.
         </BodyLong>
-        {ctx?.validUser && (
-          <>
-            <BodyLong className="mb-2">
-              Du ser også dette fordi du kan redigere denne artikkelen. Har du
-              lyst til å kontrollere innholdet nå?
-            </BodyLong>
-            <Button
-              as="a"
-              href={`https://aksel.nav.no/admin/prod/intent/edit/id=${ctx?.id}`}
-              target="_blank"
-              size="small"
-              variant="secondary-neutral"
-            >
-              Kontroller innholdet
-            </Button>
-          </>
-        )}
+        {ctx?.validUser ||
+          (true && (
```

Comment:
??

Reply:
oops

Reply:
I should find a better way to trigger conditions for website / sanity content 💭 (relying on a memory stack isn't great).